"creates a cropper and resizes the oversized image" failing in Chrome

![image](https://cloud.githubusercontent.com/assets/128755/8911303/9105e33e-3459-11e5-992b-2410f6f8ac79.png)

```
AssertionError: expected 336.350364963504 to equal 336.35
```

cc @eoger

--- a/app/tests/spec/lib/cropper.js
+++ b/app/tests/spec/lib/cropper.js
@@ -87,10 +87,9 @@ function (chai, sinon, jQuerySimulate, RouterMock, CanvasMock, p, Cropper, Ephem
             height: 7000,
             container: view.$('.cropper')
           });
-          cropper.canvas = new CanvasMock();
           assert.isTrue(resize.calledOnce);
-          assert.equal(cropper.img.width(), 240);
-          assert.equal(cropper.img.height(), 336.35);
+          assert.equal(cropper._originalHeight, 960);
+          assert.equal(cropper._originalWidth, 685);
           Cropper.prototype.resize.restore();
         });
     });

